proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+
+This process is used when selecting major initiatives to incorporate into the baseline of The Good Docs Project. In particular, it is used when deciding upon proposals to sponsor.
+
+**Version:** 1.0
+
+**Last updated:** August 2020
+
+## When do we use this process?
+
+This process is initiated periodically and typically depends upon:
+
+*   Timing of current build cycles.
+*   Sufficient bandwidth available within the community.
+*   Budget availability.
+
+Timing will typically align with the [Season of Docs](https://developers.google.com/season-of-docs) schedule.
+
+## Call for proposals
+
+On initiation of this process, a call for proposals will be sent to public forums, which will at a minimum include our [announce email list](https://thegooddocsproject.groups.io/g/announce). The announcement will include references to:
+
+*   Expectations for proposals.
+*   [Prioritization guidelines](prioritization.html).
+*   Eligibility.
+*   Associated timelines.
+*   Budget availability (if applicable).
+*   How to submit the proposal.
+
+## Eligibility
+
+*   You should be actively involved in The Good Docs Project or a related community. (Note, involvement in our community is open to all.)
+*   You will need to have won the trust of this community in order to have them back your proposal.
+*   You will need a [mentor](roles.html) before you get an application approved.
+*   You should fit with the [eligibility criteria of Season of Docs:](https://developers.google.com/season-of-docs/docs/faq#what_are_the_eligibility_requirements_for_participation)
+    *   You must be at least 18 years of age when you register.
+    *   You must be eligible to work in the country you will reside in during the program.
+    *   You must reside in a country that is not currently embargoed by the United States. See the [program rules](https://developers.google.com/season-of-docs/terms/program-rules) and the [Sanctions Programs and Country Information](https://www.treasury.gov/resource-center/sanctions/Programs/Pages/Programs.aspx) from the US Treasury.
+*   We support diversity and encourage applications and involvement from underrepresented communities.
+
+## Implementation process
+
+A variant on the following process should be considered for major initiatives.
+
+**Test the waters:**
+
+*   Float an idea in our community forums.
+*   Look for feedback, interest and engagement. 
+*   If you attract significant community interest and support, and the required effort is more than is likely to be addressed by volunteers, you might consider developing a proposal for sponsored work.
+
+**Prepare proposal:**
+
+*   Develop a proposal in conjunction with the community. Make sure it addresses our project priorities.
+*   Find a trusted mentor from our community who is willing to mentor you.
+*   Share proposal.
+
+**Review proposals:**
+
+*   Community review and discuss proposals.
+*   Community decides if the proposal is worth backing, and locks in if it is.
```

Comment:
Might need to be more specific about what "locking in" means here.

Reply:
Also: might need to have a dot point here for negotiation/determination of the stipend amount.

Reply:
> Also: might need to have a dot point here for negotiation/determination of the stipend amount.

Agree, but happy to let this persist as is until Alpha 0.1 is release.